.travis.yml

```diff
@@ -44,7 +44,7 @@ matrix:
       group: edge
       if: type = cron
       before_install:
-        - python -m pip install cython numpy scipy
+        - python -m pip install cython "numpy>=1.19.2" scipy
```

Comment:
We could technically pin the minimum numpy version on setup.py for aarch64, but I'm not sure it's a great idea (or at least it'll get messy soon).

I'm thinking we should just label aarch64 as experimental and then publish the tested versions, thoughts @MDAnalysis/coredevs ?

Reply:
If it fixes the issue, it makes sense based on the release notes for that version of NumPy: https://github.com/numpy/numpy/releases/tag/v1.19.2

There are ARM and Cython fixes mentioned.

Actually, I just took a closer look--SciPy does the exact same NumPy version pin for `aarch64` right here: https://github.com/scipy/scipy/blob/master/pyproject.toml#L26

I guess MDA could lift some of the version pins it needs from upstream `pyproject.toml` if we adopt one at some point.